required/latex-lab/latex-lab-context.dtx

```diff
@@ -106,7 +106,7 @@
 %   grouping.
 %   
 %   It would be possible to be more granular, i.e., differentiate
-%   between types of floats etc. But for now I suggest to start out
+%   between types of float etc. But for now I suggest to start out
```

Comment:
my German training always would make this "types of floats" but I'm sure you are right

Reply:
To my American ear "types of floats" sounds more natural but I have no idea which is grammatically correct